lib/mini_twitter_client.rb

```diff
@@ -0,0 +1,81 @@
+require 'faraday'
+require 'json'
+
+class Tweet
+  def initialize(name = 'name', email = 'email@email.com', message = 'message', id = nil)
+    @name = name
+    @email = email
+    @message = message
+    @id =id
+  end
+  def name
+    @name
+  end
+  def email
+    @email
+  end
+  def message
+    @message
+  end
+  def id
+    @id
+  end
+end
+
+class MiniTwitterClient
+  def initialize
+    @connection = Faraday.new(
+      url: 'http://localhost:4000',
+      params: {},
+      headers: {
+        'Accept' => 'application/json',
+        'Content-Type' => 'application/json'
+      }
+    )
+  end
+
+  def get_tweets
+    res = @connection.get('/api/tweets')
+    data = JSON.parse(res.body)['data'].map do |tweet|
+      Tweet.new(tweet['author']['name'], tweet['author']['email'], tweet['message'], tweet['id'])
+    end
```

Comment:
With keyword arguments (see the comment above) it would look like this:

```suggestion
    data = JSON.parse(res.body)['data'].map do |tweet|
      Tweet.new(
        name: tweet['author']['name'],
        email: tweet['author']['email'],
        message: tweet['message'],
        id: tweet['id']
      )
    end
```